portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,22 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private  List<String> messages;
+
+  @Override
+  public void init(){
+      messages = new ArrayList<String>();
```

Comment:
Replace all occurrences of `messages` with `this.messages`. Functionally the same as what you have but makes it more clear that `messages` is an instance variable rather than a local variable.